### PR TITLE
Fix #39: Added remove button for previously downloaded series.

### LINKED ISSUE #39: Add a button to delete cache
Currently, when a series fails to download, it is impossible to retry, since it is marked as downloaded in the TCIA Browser.

### PR DESCRIPTION
**Pull request by mehrtash**

### CODE REVIEW

**Comment by fedorov** [comment]
@mehrtash can you please take a look at the commit I added?

Did you test this branch? Doesn't it give you this error all the time? It does for me, because you are passing directory name to pydicom read. I think I fixed it, but I wanted to check why this didn't come up with your testing.

Also, can you change the indentation back to use 2 spaces? This will be consistent with all other code in QIICR, and also in Slicer.

image

**Comment by fedorov** [comment]
@mehrtash can you please respond?

File: TCIABrowser/TCIABrowser.py [hunk 1]
```diff
@@ -1410,54 +1453,64 @@ def testBrowserDownloadAndLoad(self):
     widget.getCollectionValues()
     browserWindow = widget.browserWidget
     collectionsCombobox = browserWindow.findChildren('QComboBox')[0]
-    print 'Number of collections: ',collectionsCombobox.count
-    if collectionsCombobox.count> 0:
-      collectionsCombobox.setCurrentIndex(randint(0,collectionsCombobox.count-1))
+    print
+    'Number of collections: ', collectionsCombobox.count
```
Comment (fedorov) [inline TCIABrowser/TCIABrowser.py]: @mehrtash can you please confirm you tested after your changes?

Why did you add these newlines?

Comment (fedorov) [inline TCIABrowser/TCIABrowser.py]: I see those were there in the previous code too. These newlines will result in print statement being empty, and second line treated as a comment.

**Comment by mehrtash** [comment]
@fedorov I'm working on it. I will confirm if it works well.

**Comment by fedorov** [comment]
👍  thanks!

**Comment by mehrtash** [comment]
@fedorov I think it should be fine now. I tested on a couple of cases and also run the unittest and it passed.

**Comment by fedorov** [comment]
great, thanks! I am merging
